crates/cargo-contract/src/cmd/info.rs

```diff
@@ -50,6 +54,9 @@ pub struct InfoCommand {
     /// Export the instantiate output in JSON format.
     #[clap(name = "output-json", long)]
     output_json: bool,
+    /// Display the contract's Wasm bytecode.
```

Comment:
Should we be more specific here and say "pristine" bytecode, i.e. non-instrumented? We may, for example add the possiblility to retrieve the instrumented byte code.

Reply:
It is named according to Michi comment https://github.com/paritytech/cargo-contract/pull/1013/files/234b258a2b11c309ab18ee09b35996ced1fb0c8e#r1143947838

But if you wish I can change it

Reply:
No in that case we can keep it as it is.